### ISSUE TITLE
Increasing Design Personality

### ISSUE THREAD

**Issue by chadturnbull** [northlandchurch/Website-Styles#5]
This is a more broad issue that I'm introducing, and I realize it is late in the game to be addressing something so broad, but I think we have to pursue it. I think we need to give the site a more designed aesthetic. Right now it doesn't have the personality that I think we are capable of giving it. Two examples come to mind of very similar websites to what we're building that have an added layer of design that makes the site more enjoyable: 

http://www.dominionrocket.com/
http://northlandchurch.github.io/Distributed-Church/

In both examples the site uses design to add a layer of fun and personality to the content being presented. John, you did an awesome job with the distributed church site in creating that mountain element that became a design motif through all the transitions and boxes. Dominion Rocket uses typography and unique layouts (especially that triangle near the bottom) to maintain visual interest as you navigate the site. 

We've done a really good job so far of moving this site to a clutter free, photography rich site. Those are huge strides for us. The final piece is to introduce some design elements that moves this out of purely well thought out function, to a site that is really visually engaging.

I see this as an overlay or extension of what we already have. I'm not talking about a rework or any adjustments to the layout; I'm talking about transitions between main sections, the way some of the content is presented within the grid, that kind of thing. We just need to bump up a bit of the sizzle. 

The only idea I had for a particular motif was the interconnected network elements like you had in an early wireframe. I think that could be cool, though I don't quite have it visually worked out in my mind.

**Comment by jhnwlsn** [comment]
Totally understand where you're coming from. This will most likely be a larger discussion later, but I feel like the best thing to do _at this point_ would be to put this on the back burner until I get a little further along in the project.

You know my approach well, I'd rather get you a complete system of styles rather than just a few pages that are 100% ready (with much more of the system of styles needing to be built out).

I'll label as "suspended" but won't close until we can loop back around.

**Comment by chadturnbull** [comment]
The one caveat I would add to your point is that we have a hard time evaluating the different elements without a sense for the broader design aesthetic. The layout and grid we're working from is great and we feel good about that. But, I don't feel like we have a complete design to work from. So, when we need to evaluate whether/how to do text over images, whether/how to do callouts on hero images, the way grid sections interact with one another, whether the blue stripe is better on top or below on the header, etc., we're not able to do that in context.

So, I'm fine to suspend this discussion, but there is a strong chance you'll have to do double work in the end.

**Comment by jhnwlsn** [comment]
Not sure I'm understanding... You're having trouble knowing when to use certain aesthetics? And you think having even more design will solve this issue?

**Comment by chadturnbull** [comment]
No, that's not what I'm trying to say. We evaluate all sub elements in light of the whole (that allows us to give feedback and make decisions based on what is best suited to the design of the site, not personal preference). But the whole isn't there yet. The site is still a collection of strong parts, but hasn't yet come to life. They are good parts, but until we get the full theme/personality/design built out we're just going to be nit picking element after element without actually getting at the bigger issue of the missing global design. We won't refine our way into what we're looking for. We will have to design our way there.

If we refine and refine, you'll be making constant adjustments to the design. Then you'll go back and add the layer/level of design that I'm talking about, and we will start to evaluate the component parts in light of that and you'll start getting feedback like "Would it better to use a different font for headings here?" "Let's go back to the solid black navigation because that fits the theme better." "Actually, I didn't like the text over the images before, but now I feel like it would be stronger to move those back up with a transparent banner underneath them." 

And I making sense? If not, just call me and we can talk it through.

**Comment by jhnwlsn** [comment]
Yeah, that makes more sense. Thanks for clarifying.

Let me sleep on it to make sure we go forward in the best way. What I foresee happening is a need to get back into static designs, which will ultimately add more time to production. I'm also slightly worried about adding even more stuff simply due to browser performance. There's a lot of cool stuff going on already, and I don't want to bog down browsers with just too much to handle.

Anyway, let me come back to this tomorrow/weekend and evaluate the best approach to this.

**Comment by chadturnbull** [comment]
I think that is a good plan. You'll figure out the right path forward. You nailed it with the DCv2 site reworks, and I'm confident you can do it again with this site.

**Comment by jhnwlsn** [comment]
So, let's continue to keep this "suspended" for now. I understand the caveat you brought up, but after "sleeping on it" I'm actually not too worried about that. The main reason is that the majority of stuff I'm working on right now is derived much more from the wireframes that the design comps – basically, building out all the components in their most basic form. It'll be better in process and time to circle back around to style and customization.

**Comment by wondermade** [comment]
So to clarify you'd only want design feedback now if it'd require DOM changes? Then we'll do a review of the design elements that are CSS driven at some later (but imminent!) date?

**Comment by jhnwlsn** [comment]
Yeah, I'd say that's correct. You can leave feedback on any design stuff you'd like before you forget it, I might just not get to it yet. It won't be much longer before I'll get into reviewing design details – I'm just still getting a lot of the foundation built out. If there are things that require major DOM changes, then we will need to address those separately anyway.

**Comment by chadturnbull** [comment]
Looking through the subnav page design styles one after the other is really clarifying for me. The page design for Calendar and Need Help? is stronger and less disjointed than those for Ministries and Tithe. 

These could a little something, but feel dialed to me and enjoyable to interact with and navigate:

![screen shot 2015-07-20 at 5 20 12 pm](https://cloud.githubusercontent.com/assets/10187666/8788019/99e33c62-2f05-11e5-9267-d31ac954852a.png)

![screen shot 2015-07-20 at 5 20 44 pm](https://cloud.githubusercontent.com/assets/10187666/8788024/9e32d656-2f05-11e5-9c0e-c34920679428.png)

These feel like they haven't reached what they are aiming at yet and feel disjointed and missing a general design theme:

![screen shot 2015-07-20 at 5 20 31 pm](https://cloud.githubusercontent.com/assets/10187666/8788042/c2bb1948-2f05-11e5-909b-d8f634639223.png)

![screen shot 2015-07-20 at 5 20 54 pm](https://cloud.githubusercontent.com/assets/10187666/8788045/c5c2e3fa-2f05-11e5-92d7-30bdc7595c83.png)


What I think we need to do is develop all the subnav pages and basic pages of the website around the aesthetic and style you have going with the Calendar page. Use text, subtle striping, varying fonts - weights - styles and strategic use of really good layout design to make the page enjoyable to interact with and very pleasingly functional. Then, we take the homepage, Connect, Resources, About and use those as an opportunity for more stylized design and photography rich pages in a way that leads to branding and discovery. Give these pages more personality, but still keep to the functionality of the other pages. Plus, we have to do that in a way that syncs the two styles together.

**Comment by jhnwlsn** [comment]
That's good to hear as I was thinking that building out the foundation of the site first would give you all a clearer picture into how everything will come together. For now, I'll keep pushing through and getting everything _ready_ to be more customized (as you are currently seeing) as it's hard to customize what doesn't yet exist.

We can still circle back around to customize things a little more.
